mods/default/craftitems.lua

```diff
@@ -87,16 +86,19 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 			data = stack:get_meta():to_table().fields
 		end
 
-		if data and data.owner and data.owner ~= player:get_player_name() then
+		if data and data.owner and data.owner ~= player:get_player_name()
+				and fields.title ~= "" and fields.text ~= "" then
```

Comment:
Please separate these checks. This addition allows modifying the contents by players who aren't the owner of the book.

Reply:
@SmallJoker That was the intent, when a book no text **and** title anyone could take over ownership. If the book has any text **or** title then they will not be able to modify it. Do you disagree with this intent?

Reply:
I thought it was only about erasing books where you have write permissions (i.e. are the owner of the book). Thus, my conclusions that this change introduced an unwanted behaviour.
In this case it's fine.

Reply:
> This addition allows modifying the contents by players who aren't the owner of the book.

Seems wrong to me, even if the book has no text or title.
If a book with no text or title is reverted to default:book, which i prefer, then this is not needed.